docs/docs/mlflow-3/index.mdx

```diff
@@ -138,7 +138,7 @@ with mlflow.start_run():
 
 
 traces = mlflow.search_traces(model_id=active_model_id)
```

Comment:
we cane remove `traces` here.

Reply:
Yep it'll be removed in https://github.com/mlflow/mlflow/pull/15533